src/lib/pages/backoffice/Acquisition/Order/OrderEditor/schema.js

```diff
@@ -193,9 +193,10 @@ export const schema = () => {
       status: {
         title: 'Status',
         type: 'string',
-        enum: Object.values(invenioConfig.ACQ_ORDERS.statuses).map(
+        enum: Object.values(invenioConfig.ACQ_ORDERS.statuses).sort((a, b) => a.order - b.order).map(
           (s) => s.value
         ),
+        default: Object.values(invenioConfig.ACQ_ORDERS.statuses).find((s) => s.default).value
```

Comment:
What about using [lodash _find](https://lodash.com/docs/4.17.15#find) so that if there is no `default` key it will still work?
```suggestion
        default: _get(_find(invenioConfig.ACQ_ORDERS.statuses, { 'default': true }), 'value', null)
```
to be tested! (`_find` will return `undefined` if not found). You also need to if default should have `null` or `''` when there is no default in react-jsonschema-forms.

Like this you can have only one element with `default: true`, and omit it from the others.